quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json

```diff
@@ -10,6 +10,9 @@
     },
     "administratorLoginPassword": {
       "value": "GEN-PASSWORD"
+    },
+    "sonarqubeDockerTag": {
+      "value": "GEN-SONARQUBE-IMAGE-TAG"
```

Comment:
I have not seen this image tag before, can you confirm that this deployment not only succeeded, but the app is running as expected?

Reply:
@alex-frankel Thanks for taking a look at this. I was not sure what would be the correct placeholder by looking at [this page](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#parameters-file-placeholders) hence picked this one. What I want here is `latest` as default docker tag, but possible to change as input prior deploying the template to any from [here](https://hub.docker.com/_/sonarqube/tags). What would be then correct placeholder, pehaps `latest`?

Reply:
Made it latest.

Reply:
Just to be clear, I don't know enough about this specific resource type to know if `latest` is an acceptable value. Did you check that deploying with `latest` as the parameter value deploys this as expected?

Reply:
I made the `latest` as a default value for `sonarqubeDockerTag` parameter and tested deployment. Yes, with `latest` it worked!